Resources/Prototypes/_Goobstation/Changeling/Catalog/changeling_catalog.yml

```
@@ -0,0 +1,413 @@
+# combat
+
+- type: listing
+  id: EvolutionMenuCombatArmblade
+  name: evolutionmenu-combat-armblade-name
+  description: evolutionmenu-combat-armblade-desc
+  icon: { sprite: _Goobstation/Changeling/changeling_abilities.rsi, state: armblade }
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)